CONTRIBUTING.md

```diff
@@ -61,7 +61,18 @@ Pull requests
 
 When creating pull request, please heed the following:
 
-* Base your code on the latest master branch to avoid manual merges
-* Code review may ensue in order to help shape your proposal
+* Base your code on the latest `master` branch to avoid manual merges
+* Code review may ensure in order to help shape your proposal
 * Pull request must adhere to 2-Clause BSD licensing
 * Explain the problem and your proposed solution
+* If applicable cite the issue(s) number(s) in your pull-request description,
+    for example `Fixes: #1234`, `Closes: #1234`, or `Ref: #1234`.
+* Read [README.md](./README.md) to learn about the commands to shape your code
+
+Stable release updates
+----------------------
+
+Stable versions (branches `stable/<major>.<minor>`) may be updated
+by a team member with a cherry-pick, after a pull-request is merged into the `master` branch.
+
+That's why pull-requests are always targetted to the `master` branch.
```

Comment:
I know this feels picky but in technical documentation contractions are best avoided.

```suggestion
That is why pull-requests are always targeted to the `master` branch.
```

But I'd like to restructure this differently and just say something like "After commits are included in the master branch..." in a single paragraph instead of explaining after the fact